mqtt/mqtt-edgehub/src/auth/authentication/edgehub.rs

```diff
@@ -69,7 +72,17 @@ impl Authenticator for EdgeHubAuthenticator {
         &self,
         context: AuthenticationContext,
     ) -> Result<Option<AuthId>, Self::Error> {
-        let auth_id = self.authenticate(context).await?;
+        let auth_id = (|| async {
+            info!("authenticate client");
+            let auth_id = self.authenticate(&context).await?;
+            Ok(auth_id)
+        })
+        .retry(ExponentialBackoff {
+            max_elapsed_time: Some(Duration::from_secs(60)),
```

Comment:
So it will give up after 60s? What does the exponential backoff look like (as in how many times will it retry during this time?)

Reply:
Maybe add a comment on why we are doing this and what the behavior will be like?

Reply:
It seems like it will retry 10 times in 60 seconds with the default settings. There is also some randomization, but that is by design so I think we should keep it.